recipe_sha: a45e4672216a948f7584cbdd402c3629faf173b02ad2ffd96b61b6a6ab585fb0  # The commit at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-

  The reported errors are:
  - Encountered problems while solving:
  -   - nothing provides requested bioconductor-plyranges >=1.20.0,<1.21.0
  -   - nothing provides requested bioconductor-variantannotation >=1.46.0,<1.47.0
  -   - nothing provides requested bioconductor-bsgenome >=1.68.0,<1.69.0
  -   - nothing provides icu 54.* needed by r-base-3.3.2-0
  -   - nothing provides r 3.2.2* needed by bioconductor-bsgenome.hsapiens.ucsc.hg19-1.4.0-r3.2.2_0
  -

  Leaving build/test directories:
    Work:
   /opt/conda/conda-bld/work
    Test:
   /opt/conda/conda-bld/test_tmp
  Leaving build/test environments:
    Test:
  source activate  /opt/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
    Build:
  source activate  /opt/conda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 244, in solve_for_action
      t = self.solve(specs)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 234, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - bioconductor-plyranges >=1.20.0,<1.21.0
   - r-tidyr >=1.2.0
   - r-checkmate >=2.0.0
   - r-changepoint >=2.2.3
   - r-ggplot2 >=3.3.5
   - bioconductor-maftools >=2.16.0,<2.17.0
   - bioconductor-biocparallel >=1.34.0,<1.35.0
   - bioconductor-s4vectors >=0.38.0,<0.39.0
   - bioconductor-variantannotation >=1.46.0,<1.47.0
   - bioconductor-genomeinfodb >=1.36.0,<1.37.0
   - r-rlang >=1.0.2
   - bioconductor-genomicranges >=1.52.0,<1.53.0
   - r-base 4.3.*
   - bioconductor-bsgenome.hsapiens.ucsc.hg19 >=1.4.0,<1.5.0
   - r-ggtext >=0.1.1
   - bioconductor-iranges >=2.34.0,<2.35.0
   - r-tibble >=3.1.6
   - r-dplyr >=1.0.8
   - bioconductor-bsgenome.hsapiens.ucsc.hg38 >=1.4.0,<1.5.0
   - r-changepoint.np >=1.0.3
   - r-rdpack >=2.3.1
   - bioconductor-bsgenome >=1.68.0,<1.69.0
   - bioconductor-biobase >=2.60.0,<2.61.0

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - nothing provides requested bioconductor-plyranges >=1.20.0,<1.21.0
  -   - nothing provides requested bioconductor-variantannotation >=1.46.0,<1.47.0
  -   - nothing provides requested bioconductor-bsgenome >=1.68.0,<1.69.0
  -   - nothing provides icu 54.* needed by r-base-3.3.2-0
  -   - nothing provides r 3.2.2* needed by bioconductor-bsgenome.hsapiens.ucsc.hg19-1.4.0-r3.2.2_0
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2038, in build
      output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 787, in expand_outputs
      for (output_dict, m) in deepcopy(_m).get_output_metadata_set(permit_unsatisfiable_variants=False):
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 2524, in get_output_metadata_set
      conda_packages = finalize_outputs_pass(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 884, in finalize_outputs_pass
      fm = finalize_metadata(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 547, in finalize_metadata
      build_unsat, host_unsat = add_upstream_pins(m,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 409, in add_upstream_pins
      host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(m, 'host',
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 374, in _read_upstream_pin_files
      deps, actions, unsat = get_env_dependencies(m, env, m.config.variant,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 131, in get_env_dependencies
      actions = environ.get_install_actions(tmpdir, tuple(dependencies), env,
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("bioconductor-plyranges[version='>=1.20.0,<1.21.0']"), MatchSpec("icu=54"), MatchSpec("bioconductor-bsgenome[version='>=1.68.0,<1.69.0']"), MatchSpec("r=3.2.2"), MatchSpec("r-base==3.3.2=0"), MatchSpec("bioconductor-variantannotation[version='>=1.46.0,<1.47.0']"), MatchSpec("bioconductor-bsgenome.hsapiens.ucsc.hg19==1.4.0=r3.2.2_0")}
# Last 100 lines of the build log.
